recipes-domu/domu-image-android/domu-image-android.bbappend

```diff
@@ -1,30 +1,20 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
-FILESEXTRAPATHS_prepend := "${THISDIR}/../../recipes-domx:"
+# Most functionality and settings are defiend in xt-images.
+# Only some minimal tuning is required by environment variables.
 
-# we need MACHINEOVERRIDES from DomD build
-do_configure[depends] += "domd-image-weston:do_domd_install_machine_overrides"
+# We use SRC_URI provided by meta-xt-images
 
-SRC_URI = " \
-    repo://github.com/xen-troops/manifests;protocol=https;branch=master;manifest=prod_devel/domu_android_host_tools.xml;scmdata=keep \
-"
+# Android itself doesn't care about hardware, but graphics does.
+# That's why we need to check that we are building for supported hardware.
+python __anonymous () {
+    machines = d.getVar("MACHINEOVERRIDES", expand=True)
+    if ("r8a7795-es3" not in machines):
```

Comment:
So, are we going to extend this list over the time adding new supported platforms then?

Reply:
Only if we will provide support for new platforms.